src/yaml/updateLayerMetadata.yaml

```diff
@@ -93,28 +93,28 @@ components:
           maximum: 8000
           format: double
           description: max (rounded) resolution of layer in meters/pixel
-        accuracyCE90:
+        minHorizontalAccuracyCE90:
           type: number
-          description: Horizontal Accuracy
-        sensorType:
+          minimum: 0.01
```

Comment:
same validation is missing on field decoration level ( see an example in *resolution* decoration)